packages/react/test/specs/components/Dropdown/Dropdown-test.tsx

```diff
@@ -357,5 +865,160 @@ describe('Dropdown', () => {
+      const wrapper = mountWithProvider(
+        <Dropdown onSearchQueryChange={onSearchQueryChange} search items={items} />,
+      )
+
+      wrapper
+        .find(`input.${DropdownSearchInput.slotClassNames.input}`)
+        .simulate('click')
+        .simulate('change', { target: { value: 'test' } })
+        .simulate('keydown', { keyCode: keyboardKey.Escape, key: 'Escape' })
+
+      expect(onSearchQueryChange).toBeCalledTimes(2)
+      expect(onSearchQueryChange).toHaveBeenLastCalledWith(
+        null,
+        expect.objectContaining({
+          searchQuery: '',
+        }),
+      )
+    })
+
+    it('is set to empty when item is selected in multiple search', () => {
+      const itemSelectedIndex = 2
+      const onSelectedChange = jest.fn()
+      const wrapper = mountWithProvider(
+        <Dropdown onSelectedChange={onSelectedChange} search multiple items={items} />,
+      )
+
+      wrapper.find(`span.${Dropdown.slotClassNames.toggleIndicator}`).simulate('click')
+      wrapper
+        .find(`li.${Dropdown.slotClassNames.item}`)
+        .at(itemSelectedIndex)
+        .simulate('click')
+
+      expect(onSelectedChange).toHaveBeenLastCalledWith(
+        null,
+        expect.objectContaining({
+          value: [items[itemSelectedIndex]],
+          searchQuery: '',
+        }),
+      )
+    })
+  })
+
+  describe('toggleIndicator', () => {
+    it('closes the open menu on click', () => {
+      const onOpenChange = jest.fn()
+      const wrapper = mountWithProvider(<Dropdown onOpenChange={onOpenChange} items={items} />)
+
+      wrapper.find(`button.${Dropdown.slotClassNames.triggerButton}`).simulate('click')
+      wrapper.find(`span.${Dropdown.slotClassNames.toggleIndicator}`).simulate('click')
+
+      expect(onOpenChange).toBeCalledTimes(2)
+      expect(onOpenChange).toHaveBeenLastCalledWith(
+        null,
+        expect.objectContaining({
+          open: false,
+        }),
+      )
+    })
+
+    it('opens the menu and moves focus to list in selection mode', () => {
+      const wrapper = mountWithProvider(<Dropdown items={items} />)
+
+      wrapper.find(`span.${Dropdown.slotClassNames.toggleIndicator}`).simulate('click')
+
+      expect(document.activeElement).toEqual(
+        wrapper.find(`ul.${Dropdown.slotClassNames.itemsList}`).getDOMNode(),
+      )
+    })
+
+    it('opens the menu and moves focus to input in search mode', () => {
+      const wrapper = mountWithProvider(<Dropdown items={items} search />)
+
+      wrapper.find(`span.${Dropdown.slotClassNames.toggleIndicator}`).simulate('click')
+
+      expect(document.activeElement).toEqual(
+        wrapper.find(`input.${DropdownSearchInput.slotClassNames.input}`).getDOMNode(),
+      )
+    })
+  })
+
+  describe('moveFocusOnTab', () => {
+    it('keeps focus on trigger button when not passed', () => {
+      const wrapper = mountWithProvider(
+        <>
+          <Dropdown multiple items={items} />,<div tabIndex={0} id="give-me-focus" />
```

Comment:
```suggestion
          <Dropdown multiple items={items} />
          <div tabIndex={0} id="give-me-focus" />
```

nit: comma looks very confusing there, is it necessary?

Reply:
just a mistake. I will remove this whole suite, since Tab is broken and I cannot consider these tests to be of any value.